meshroom/nodes/aliceVision/MaskProcessing.py

```diff
@@ -0,0 +1,74 @@
+__version__ = "1.0"
+
+from meshroom.core import desc
+from meshroom.core.utils import DESCRIBER_TYPES, VERBOSE_LEVEL
+
+import os.path
+
+class MaskProcessingNodeSize(desc.DynamicNodeSize):
+    """
+    MaskProcessingNodeSize expresses a dependency to multiple input attributess to define
+    the size of a Node in terms of individual tasks for parallelization.
+    """
+    def __init__(self, param):
+        self._params = param
+
+    def computeSize(self, node):
+        
+        size = 0
+
+        for input in node.attribute(self._params).value:
+            paramName = input.getFullName()
+            param = node.attribute(paramName)
+            if param.isLink:
+                size = max(size, param.getLinkParam().node.size)
+        
+        return size
+
+
+class MaskProcessing(desc.AVCommandLineNode):
+    commandLine = 'aliceVision_maskProcessing {allParams}'
+    size = MaskProcessingNodeSize("inputs")
+
+    category = 'Utils'
+    documentation = '''
+    Perform operations on a list of masks with the same names
+    '''
+
+    inputs = [
+        desc.ListAttribute(
+            elementDesc=desc.File(
+                name="input",
+                label="Input Directory",
+                description="A directory with a set of mask.",
+                value="",
+            ),
+            name="inputs",
+            label="Input directories",
+            description="A set of directories containing masks with the same names.",
+            exposed=True,
+        ),
+        desc.ChoiceParam(
+            name="operator",
+            label="Operator",
+            description="Operator: Logic operator",
+            value="and",
+            values=["or", "and", "not"],
+        ),
+        desc.ChoiceParam(
+            name="verboseLevel",
+            label="Verbose Level",
+            description="Verbosity level (fatal, error, warning, info, debug, trace).",
+            values=VERBOSE_LEVEL,
+            value="info",
+        )
+    ]
+
+    outputs = [
+        desc.File(
+            name="output",
+            label="Output",
+            description="Path to the output directory.",
+            value=desc.Node.internalFolder,
+        )
```

Comment:
Need to expose the output masks to the Viewer.
Here is an example: https://github.com/alicevision/Meshroom/blob/develop/meshroom/nodes/aliceVision/ImageSegmentation.py#L86

Reply:
> Need to expose the output masks to the Viewer. Here is an example: https://github.com/alicevision/Meshroom/blob/develop/meshroom/nodes/aliceVision/ImageSegmentation.py#L86

How do i do that ? I have no clue about the masks names as it may come from various sources.